README.md

```diff
@@ -1,38 +1,19 @@
-# Hellofresh - Golang Reading List
+# HelloFresh - Golang Guides
 
-Welcome to our golang reading list. If you are new to golang we recomend going through our getting started links.
+Welcome to HelloFresh guides for Go projects and developers.
 
-## Getting Started
-- https://www.golang-book.com/books/intro
-- https://tour.golang.org/welcome/1
-- https://golang.org/doc/
-- https://golang.org/doc/effective_go.html
+- [Best Practices](./best_practices.md)
+- [Code Style](./code_style.md)
+- [Directory Structure](./directory_structure.md)
+- [Packages](./packages.md)
+- [Projects](./projects.md)
+- [Reading List](./reading_list.md)
+- [Tools](./tools.md)
 
-## Tips
-- https://gobyexample.com/
+## Contributing
 
-## Advanced Topics
-- http://peter.bourgon.org/go-best-practices-2016/
-- https://divan.github.io/posts/avoid_gotchas/
-- https://medium.com/@Martynas/formatting-date-and-time-in-golang-5816112bf098
+Use common sense and Pull Requests for the win, and may the --force be with you!
 
-## Books
-- https://www.safaribooksonline.com/library/view/the-go-programming/9780134190570/
-- https://www.safaribooksonline.com/library/view/go-programming-blueprints/9781786468949/
-- https://www.safaribooksonline.com/library/view/go-in-action/9781617291784/
-- https://www.safaribooksonline.com/library/view/go-in-practice/9781633430075/
-
-## Talks / Videos
-- https://www.youtube.com/playlist?list=PL64wiCrrxh4Jisi7OcCJIUpguV_f5jGnZ (Just for func by Francesc Campoy) 
-- https://www.udemy.com/learn-how-to-code Go for beginners by Todd McLeod  (Paid) 
-- https://www.udemy.com/go-programming-language/ Web Development with Go by Todd McLeod (Paid) 
-- https://www.youtube.com/user/toddmcleod 
-- https://www.safaribooksonline.com/library/view/introducing-go/9781491941997/ Introducing Go
-
-## Blogs 
-- https://www.goinggo.net/ (Bill Kennedy)
-- https://dave.cheney.net/ 
-
-## Podcasts
-- https://changelog.com/gotime 
+## Support and Feedback
 
+If you have a question or just want to start a holy war, feel free to file an issue or ask in `#clan-golang` channel in Slack.
```

Comment:
Not sure about this reference in a public repo 🤔

Reply:
@bastiankoetsier fixed